tests/publishing/templates/validations.test.js

```diff
@@ -7,44 +7,77 @@ function loginAndClickTemplates() {
+  });
 
-  // it('the lotion', () => {
-  //   expect(true).to.be.true;
-  // });
 
-  // it('on', () => {
-  //   expect(true).to.be.true;
-  // });
+  it('should refuse a new template with a name and invalid path', () => {
+    return common.waitAndClick('//button[@id="new_tmpl"]')
+      .then(() => browser.waitForExist('//*[@id="editTemplate"]'))
+      .then(() => browser.waitForExist('//*[@id="editTemplate"]//*[@id="name"]'))
+      .then(() => browser.setValue('//*[@id="editTemplate"]//*[@id="name"]', 'test name for test purposes only'))
+      .then(() => browser.setValue('//form[@id="newTmplForm"]//input[@id="path"]', 'invalid input'))
+      .then(() => common.waitAndClick('//*[@id="subBtn"]'))
+      .then(() => browser.waitForExist('//*[@id="admin-alert"]', 10000))    
+      .then(() => common.waitAndClick('//*[@id="newTmplForm"]//button[2 and text()="Cancel"]'));
+  });
 
 
-  // describe('Path', () => {
-  //   afterEach(() => somehowClearThePathField());
-  //   it('should not allow the word monkey', () => {
-  //     expect(true).to.be.true;
-  //   });
+  it('should refuse a blank add template form', () => {
+    return common.waitAndClick('//*[@id="add_tmpl"]')
+      .then(() => common.waitAndClick('//*[@id="subBtn"]'))
+      .then(() => browser.waitForExist('//*[@id="name-message"]', 10000));
+  });
 
-  //   it('should not allow the characters $%^^&**^%', () => {
-  //     expect(true).to.be.false;
-  //   });
-  // })
+  it('should refuse an add template form without a path', () => {
+    return common.waitAndClick('//*[@id="add_tmpl_form"]/div[3]/button[2 and text()="Cancel"]')
+      .then(() => common.waitAndClick('//*[@id="add_tmpl"]'))
+      .then(() => browser.setValue('//*[@id="name"]', 'All Hail Slithis!'))
+      .then(() => common.waitAndClick('//*[@id="subBtn"]'))
+      .then(() => browser.waitForExist('//*[@id="path-message><i class="]', 10000))
+      .then(() => common.waitAndClick('//*[@id="add_tmpl_form"]//button[2 and text()="Cancel"]'));
+  });
 
-});
+  it('should refuse an add template form without a name and an invalid path', () => {
+    return common.waitAndClick('//*[@id="add_tmpl"]')
+      .then(() => browser.setValue('//*[@id="path"]', 'All Hail Slithis!'))
+      .then(() => common.waitAndClick('//*[@id="subBtn"]'))
+      .then(() => browser.waitForExist('//*[@id="path-message"]', 10000))      
+      .then(() => common.waitAndClick('//*[@id="add_tmpl_form"]//button[2 and text()="Cancel"]'));
+  });
 
+  it('should refuse an add template form with a name and an invalid path', () => {
+    return common.waitAndClick('//*[@id="add_tmpl"]')
+      .then(() => browser.setValue('//*[@id="name"]', 'All Hail Slithis!'))
+      .then(() => browser.setValue('//*[@id="path"]', 'All Hail Slithis!'))
+      .then(() => common.waitAndClick('//*[@id="subBtn"]'))
+      .then(() => browser.waitForExist('//*[@id="path-message"]', 10000));
+  });
 
+});
```

Comment:
it only had a before not beforeEach in this one so it was not logging in and out between each test.